misc/config_tools/schema/types.xsd

```diff
@@ -192,21 +221,35 @@ device in VM . Set in hex.</xs:documentation>
 </xs:simpleType>
 
 <xs:complexType name="IVSHMEMRegionType">
-  <xs:sequence>
+  <xs:all>
+    <xs:element name="NAME">
+      <xs:annotation acrn:title="Name">
+        <xs:documentation>Name of the shared memory region.</xs:documentation>
+      </xs:annotation>
+      <xs:simpleType>
+	<xs:restriction base="xs:string">
+	  <xs:pattern value="\w+" />
```

Comment:
Is there a length restriction for this name?

Reply:
Not yet. This name will be used solely to name macros that identify those for different regions, as C (as supported by recent GCC) does not impose a hard limit on the length of macro names.